### DISCUSSION TITLE
Interesting go guardian glitch with chrome://restart and my dns

### DISCUSSION THREAD

**Discussion by 88411** [3kh0/ext-remover#1392] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Bruh this is literally a new method on how to disable go guardian. Why does litterly no one care 😭

Story time:
I was on my friends Chromebook helping him bypass goguardian so he could access websites with the phrase "game" for his research project. I was simply going to use link preview exploit so that he could read through the article. I also had him on my dns server from my previous post as well. When I enabled the flag and restarted the Chromebook all of sudden go goguardian stopped working. 

After some testing I realized that if you are on the dns that blocks go guardian it does nothing if it is installed because it still pops up a block page even if I can't load it properly. However if you type in chrome://restart with the dns enabled, go guardian just forgets what to block? I'm not completely sure how it works but it seems to make it just forgets how to work.Maybe it's memory or cache gets wiped and bc you are on the DNS, it can't redownload the policies that tell it what to block.

But that's just my best guess. I just discovered this fyi and I will keep testing. Just wanted to let everyone know about this.

Forgot to include DNS:

The dns server is 45.90.28.127 fyi and the syncing link is

https://link-ip.nextdns.io/f68489/5a4308a7247a0cbc

If the dns doesn't work just click the link or just set up the dns on your router

**Comment by 88411** [comment]
I feel stupid but this also works with restarting the Chromebook regularly.

**Comment by 88411** [comment]
This could prove useful for developing an exploit because all you would have to do is be on a dns and click a bookmark and then go guardian stops working

**Comment by ghost** [comment]
Hmmm I really need this so I'll try to develop it

**Reply by 88411** [reply]
The dns server is 45.90.28.127 fyi and the syncing link is

https://link-ip.nextdns.io/f68489/5a4308a7247a0cbc

If the dns doesn't work just click the link or just set up the dns on your router

**Reply by 88411** [reply]
Let me know if you need anymore help. I am pretty familiar with these dns related exploits.

**Comment by Perplexedykwhat** [comment]
prob should get this to a contributer rn

**Comment by Luigi-Master** [comment]
I would have tried this but go guardian dosen't block my webpages, it only looks what Im doing on my Chromebook. But if this works, this could be a huge exploit and probably be in the repository.

**Reply by 88411** [reply]
This will prevent you from being stalked as well as the dns blocks all go guardian traffic. Matter of fact if all you need is to not be stalked then you don't have to any of the other steps. Just set your dns to 45.90.28.127 and sync it with the link and your good to go

**Comment by akabutnicer** [comment]
yeah, thats what i found out via my goguardian fever exploit. restarting chrome os clears goguardian's cache, which typically further compliments an exploit.

**Reply by 88411** [reply]
What is the go guardian fever exploit

**Reply by akabutnicer** [reply]
guys what is the go guardian fever exploit

**Comment by schoolexploitkid** [comment]
tbh, just unenroll with br1ck or icarus (in a few days), or my unenroll method if u have like $20 https://github.com/schoolexploitkid/De-Enterprise-unenrollment

**Reply by 88411** [reply]
Yeah Ik I could unenroll very easily. I have even done pencil method but I'm in pretty big trouble with my school regarding chroembooks and removing go guardian so this method is a lot better bc you can just get off the dns to undo it. 

But yea you have a point.

**Reply by 88411** [reply]
What is icacus or whatever about. I've been hearing about it since a few months ago

**Reply by FunnyCorgi** [reply]
@schoolexploitkid this isn't an exploit exactly(it still sort of is, just not exactly exploiting a bug). This is just swapping the emmc ssd between the two laptops. If I were you, I'd **make sure that you have in the instructions to make sure the other chromebook has already installed chromeos on it/chromeos is not on the install screen when it turns on, because otherwise it will enroll based off of the FWMP TMP space on the chip on the motherboard in the school chromebook.**

**Reply by Charlieis1nsan3** [reply]
> What is icacus or whatever about. I've been hearing about it since a few months ago


its a new unenrollment method

**Comment by ghost** [comment]
GUYS IT WORKS THANKS OMG IT WORKS I JUSTTRYED IT AND I FREKIN UNBLOCKED EVEYTHING

**Reply by 88411** [reply]
Ik xD

**Reply by 88411** [reply]
I posted a kinda official tutorial for it

**Comment by Charlieis1nsan3** [comment]
Noice, but I have securly, but I found a glitch cuz my IT is retarded as FUCK, chrome://settings is blocked, but when i press ctrl+h and click delete browsing data it just takes me to chrome://settings and I click out of the popup and im in settings 🤣

**Comment by JWU9292** [comment]
the respretory shows bad request.

**Comment by JWU9292** [comment]
pls help the link shows bad request.
